skyportal/models.py

```diff
@@ -3448,6 +3454,355 @@ class SourceNotification(Base):
     level = sa.Column(sa.String(), nullable=False)
 
 
+class GcnNotice(Base):
+    """Records of ingested GCN notices"""
+
+    update = delete = AccessibleIfUserMatches('sent_by')
+
+    ivorn = sa.Column(sa.String, doc='Unique identifier of VOEvent')
```

Comment:
Sounds like this should be 
```suggestion
    ivorn = sa.Column(sa.String, unique=True, index=True, doc='Unique identifier of VOEvent')
```